src/Akeneo/Nelson/ConsoleLogger.php

```diff
@@ -34,36 +34,30 @@ public static function getSubscribedEvents()
      */
     public function preNelsonPull(GenericEvent $event)
     {
-        $this->writeProcessing(sprintf(
-            'Pulling translations of branch <bold>%s</bold> from Crowdin to Github',
-            $event->getArgument('branch')
-        ));
+        $this->writeProcessing(Events::PRE_NELSON_PULL, $this->getTranslationParams($event->getArguments()));
     }
 
     /**
      * @param Event $event
      */
     public function postNelsonPull(Event $event)
     {
-        $this->writeSuccess('Translation pulled!');
+        $this->writeSuccess(Events::POST_NELSON_PULL);
     }
 
     /**
      * @param GenericEvent $event
      */
     public function preNelsonPush(GenericEvent $event)
     {
-        $this->writeProcessing(sprintf(
-            'Pushing translation keys of branch <bold>%s</bold> from Github to Crowdin',
-            $event->getArgument('branch')
-        ));
+        $this->writeProcessing(Events::PRE_NELSON_PUSH, $this->getTranslationParams($event->getArguments()));
```

Comment:
getTranslationParams could be done inside the method writeProcessing to group the "translation concern" in a single place